src/lando/settings.py

```diff
@@ -138,6 +139,28 @@
     BASE_DIR / "static_src",
 ]
 
+STATICFILES_FINDERS = [
+    "django.contrib.staticfiles.finders.FileSystemFinder",
+    "django.contrib.staticfiles.finders.AppDirectoriesFinder",
+    "compressor.finders.CompressorFinder",
+]
+
+COMPRESS_PRECOMPILERS = (
+    ("text/x-scss", "django_libsass.SassCompiler"),
+)
+
+COMPRESS_FILTERS = {
+    'css':[
+        'compressor.filters.css_default.CssAbsoluteFilter',
+        'compressor.filters.cssmin.rCSSMinFilter',
+    ],
+    'js':[
+        'compressor.filters.jsmin.JSMinFilter',
+    ]
+}
+
+COMPRESS_ENABLED = True
```

Comment:
Add some comments explaining these.

Reply:
This replaces flask-assets, along with all the asset-building functionality in the legacy lando-ui repo. Will add that to the commit message.